src/styles/Panel.css

```diff
@@ -1,82 +1,148 @@
+  display: flex;
+  height: 100%;
+  box-sizing: border-box;
+  width: 100%;
+  background-color: #272134;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  box-sizing: content-box;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  position: relative;
 }
 
-.panel-collapse-button{
-	display:flex;
-	padding:1rem;
-    color:#dddcdf;
-	width:100%;
-	justify-content: flex-end;
-	cursor:pointer;
+.panel-collapse-button {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  color: #dddcdf;
+  cursor: pointer;
+  font-size: 35px;
 }
 
-.panel-content{
-	display:flex;
-	widtH:100%;
-	min-width:300px;
-	flex-direction:column;
-	align-items:center;
-	justify-content:flex-start;
-	overflow-y:auto;
-	overflow-x:hidden;
+.panel-content {
+  display: flex;
+  width: 100%;
+  padding-top: 40px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  overflow-y: auto;
+  overflow-x: hidden;
+  box-sizing: content-box;
 }
 
-.panel-image{
-	border-radius:500px;
-	border: 1px solid #dddcdf;
-	width:30%;
-	min-width:225px;
-	/*background:#000;*/ /*if you leave this out, transparent profile pictures will show the background behind it*/
+.panel-image {
+  border-radius: 500px;
+  border: 1px solid #dddcdf;
+  width: 205px;
+  height: 200px;
+  /*background:#000;*/ /*if you leave this out, transparent profile pictures will show the background behind it*/
 }
 
-.panel-name{
-	width:25vw;
-	float:middle;
-	text-align:center;
-	padding:27px 0 5px;
-	font-size:2.5rem;
-	font-weight: bold;
-    color:#dddcdf;
+.panel-name {
+  float: middle;
+  text-align: center;
+  margin: 0px 10px;
+  max-width: 90%;
+  padding: 20px 0 5px;
+  font-size: 2rem;
+  font-weight: bold;
+  word-wrap: break-word;
+  color: #dddcdf;
 }
 
-.panel-options{
-	width:100%;
-    list-style-position: inside;
-	list-style-type:square;
-	font-size:2.5rem;
-	text-align:center;
-	justify-content: flex-start;
+.panel-options-item {
+  border: 2px solid transparent;
+  border-radius: 10px;
+  background-color: #857e8f;
+  padding-left: 5px;
+  padding-right: 5px;
+  color: rgb(133, 129, 129);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  position: relative;
+  margin-bottom: 20px;
 }
 
-.panel-options-list{
-	display:inline-block;
-	padding:0px;
-	margin-top: 25px;
+.panel-options-item:hover {
+  border: 2px solid white;
+  border-radius: 10px;
 }
 
-.panel-options-item{
-	font-weight:normal;
-	font-size:2.5rem;
-	width:fit-content;
-	margin:0px;
-    color:#dddcdf;
-	justify-content: flex-start;
+.panel-options-item-disabled {
+  border: 2px solid transparent;
+  border-radius: 10px;
+  background-color: #857e8f;
+  padding-left: 5px;
+  padding-right: 5px;
+  color: rgb(133, 129, 129);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  position: relative;
+  margin-bottom: 20px;
+}
+.panel-item-content {
+  position: relative;
+  font-size: 33px;
```

Comment:
should we hard code w/ pixels or use em for font sizes, thoughts @connorbo97

Reply:
Still unsure of the px vs em relationship and when to use em tbh. Can you explain the main reason to switch?